packages/types/tsup.config.ts

```diff
@@ -1,15 +0,0 @@
-import { defineConfig } from 'tsup';
```

Comment:
What's this file?

Reply:
It was for a bundler I was testing with the types package but didn't end up needing. Must've missed deleting it when I added the types package, so removing now as its not actually used.